src/main/resources/db/migration/V14__deltaker-person-id.sql

```diff
@@ -0,0 +1,12 @@
+alter table nav_bruker rename column personid to person_id;
+
+alter table deltaker add column person_id uuid references nav_bruker;
+
+update deltaker
+set person_id = nb.person_id
+from nav_bruker nb
+where deltaker.personident = nb.personident;
```

Comment:
Jeg har lest inn all personalia for deltakerne som manglet det manuelt, og må kanskje gjøre det en gang til før det merges.